File: aj_flutter_webview/ios/Classes/AjFlutterWebviewPlugin.m
#import "AjFlutterWebviewPlugin.h"
#import "FlutterWebView.h"

@implementation AjFlutterWebviewPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
    AJWebViewFactory *webviewFactory = [[AJWebViewFactory alloc] initWithMessenger:registrar.messenger];
    [registrar registerViewFactory:webviewFactory withId:@"aj_flutter_webview"];
}
@end

File: aj_flutter_webview/android/src/main/java/com/plus/anji/ajflutterwebview/AjFlutterWebviewPlugin.java
package com.plus.anji.ajflutterwebview;

import android.app.Activity;
import android.content.Context;
import android.content.Intent;
import android.graphics.Point;
import android.os.Build;
import android.view.Display;
import android.webkit.CookieManager;
import android.webkit.ValueCallback;
import android.widget.FrameLayout;

import java.util.Map;

import io.flutter.plugin.common.MethodCall;
import io.flutter.plugin.common.MethodChannel;
import io.flutter.plugin.common.MethodChannel.MethodCallHandler;
import io.flutter.plugin.common.MethodChannel.Result;
import io.flutter.plugin.common.PluginRegistry;
import io.flutter.plugin.common.PluginRegistry.Registrar;

/** AjFlutterWebviewPlugin */
public class AjFlutterWebviewPlugin {
  /** Plugin registration. */
  public static void registerWith(Registrar registrar) {
    registrar
            .platformViewRegistry()
            .registerViewFactory(
                    "aj_flutter_webview", new WebViewFactory(registrar.messenger()));
  }
}